docs/source/jupyter_lite_config.json

```diff
@@ -48,6 +49,7 @@
       "https://files.pythonhosted.org/packages/py3/j/jsonschema/jsonschema-4.4.0-py3-none-any.whl",
       "https://files.pythonhosted.org/packages/py3/j/jupyter-client/jupyter_client-7.1.2-py3-none-any.whl",
       "https://files.pythonhosted.org/packages/py3/j/jupyter-core/jupyter_core-4.9.2-py3-none-any.whl",
+      "https://files.pythonhosted.org/packages/py3/j/jupyterlab_github-3.0.1-py3-none-any.whl",
```

Comment:
this probably to do anything, as it's currently not feasible for serverextensions to run in browser kernels...

Reply:
Oh, duh, right. Well, good thing the github extension works in pure browser mode too, I think...